docs/snippets/proc-deploying-function.md

```diff
@@ -0,0 +1,58 @@
+<!-- Snippet used in the following topics:
+- /docs/getting-started/build-run-deploy-func.md
+- /docs/functions/build-run-deploy-func.md
+-->
+The `deploy` command uses the function project name and the image registry name to construct a fully qualified image name for your function.
+
+=== "func"
+
+    Deploy the function by running the command inside the project directory:
+
+    ```bash
+    func deploy --registry <registry>
+    ```
+
+=== "kn func"
+
+    Deploy the function by running the command inside the project directory:
+
+    ```bash
+    kn func deploy --registry <registry>
+    ```
+
+!!! Success "Expected output"
+    ```{ .bash .no-copy }
+        🙌 Function image built: <registry>/hello:latest
+        ✅ Function deployed in namespace "default" and exposed at URL:
+        http://hello.default.127.0.0.1.sslip.io
+    ```
+
+You can verify that your function has been successfully deployed by using the `invoke` command and observing the output:
```

Comment:
This also works locally, and I wonder if it would be useful to have similar text in the "Running a function" section.